Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -117,7 +117,17 @@ function Cesium3DTilesInspector(container, scene) {
   tilesetPanelContents.appendChild(createCheckbox("Enable Picking", "picking"));
 
   displayPanelContents.appendChild(createCheckbox("Colorize", "colorize"));
-  displayPanelContents.appendChild(createCheckbox("Wireframe", "wireframe"));
+  displayPanelContents.appendChild(
+    createCheckbox("Wireframe", "wireframe", "hasEnabledWireframe")
+  );
+
+  //Create warning text when the Wireframe checkbox is disabled
+  const warningText = document.createElement("p");
+  warningText.setAttribute("data-bind", `visible: !hasEnabledWireframe`);
+  warningText.setAttribute("style", "margin: 5px 0 0 0; padding: 0 0 0 20px");
```

Comment:
I would suggest moving this to a CSS class in `Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.css`.